onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2190,18 +2190,41 @@ def aten_dist(self: TensorType, other: TensorType, p: float = 2.0) -> TensorType
     raise NotImplementedError()
 
 
-@torch_op(("aten::div", "aten::div.Tensor"))
+@torch_op(
+    (
+        "aten::div",
+        "aten::div.Tensor",
+        "aten::div.Scalar",
+        # When rounding_mode is None, performs a true division
+        # https://pytorch.org/docs/stable/generated/torch.div.html
```

Comment:
If dispatcher expected to filter any attribute with None?

Reply:
I would consider this to be a better match I think? Any suggestions?

Reply:
I do think the dispatcher should strip None keyword args

Reply:
No I think that makes sense. It's just we are altering attributes, and `param_schema` matching is diverged from the inputs/attributes sent into OnnxFunction. It's like there are many indications around dispatching/OnnxFunction `param_schema`. And it's not good for debugging.

Dispatcher alters inputs/attributes with hidden assumptions, but never return the altered inputs/attributes. So in OnnxFunction perspective, it runs directly on that dispatched function with attributes it doesn't need (won't error).